tasks/install.yml

```diff
@@ -1,18 +1,12 @@
 ---
 
- #- name: Disable default postgres packages
- # copy:
- #   src: exclude-default-postgres.repo
- #   dest: /etc/yum.repos.d/exclude-default-postgres.repo
- #   owner: root
- #   group: root
- #   mode: 0644
-
-- name: Disable default postgres packages
-  yum:
-    name: "postgresql" 
-    state: absent
-
+ - name: Disable default postgres packages
+  copy:
+    src: exclude-default-postgres.repo
+    dest: /etc/yum.repos.d/exclude-default-postgres.repo
```

Comment:
Since this block isn't working I think we do want to make sure that we're deleting it.

Reply:
I deleted the block, and re-added the block with "state: absent", then pushed the changes to my PR branch. The commit is now included in this request.